.github/workflows/production_pipeline.yml

```diff
@@ -180,4 +180,5 @@ jobs:
         cf set-env $APP_NAME IMPORT_PAAS_INSTANCE $IMPORT_PAAS_INSTANCE
         cf set-env $APP_NAME EXPORT_PAAS_INSTANCE $EXPORT_PAAS_INSTANCE
         cf set-env $APP_NAME SENTRY_DSN $SENTRY_DSN
+        cf set-env $APP_NAME RAILS_LOG_TO_STDOUT true
```

Comment:
Does make any practical difference?

Reply:
I'm wondering if you're just adding it for explicitness whether it might be unexpected that removing it would just set it back to `enabled` rather than `false` and whether it might be confusing that the default is `enabled` and this is `true` (do they do something different?). Maybe I'm overthinking this...